parameters/world_modeling/world_model_gmapping.yaml

```diff
@@ -1,21 +1,8 @@
-world_name: empty
+include: world_model_gmapping_sim.yaml
```

Comment:
It seems really odd to me to include sim stuff in a non-sim file

Reply:
Yep, but the robot config is an extension of the sim config. (Difference most of the times is the rgbd obstacle avoidance.)